frontend/uddhar-frontend/src/coordinator/components/EmergencyMsg.jsx

```
@@ -1,24 +1,25 @@
-      </form>
-    </>
+    <form action="">
+      <div className="flex flex-col">
+        <label htmlFor="message">Send Emergency Message</label>
+        <textarea
+          name="message"
+          id="message"
+          cols="30"
+          rows="5"
+          className="border-2 border-black p-[5px] rounded-md mt-5"
+        ></textarea>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)